### ISSUE TITLE
.NET 5.0 support

### ISSUE THREAD

**Issue by SteveSandersonMS** [jsakamoto/Toolbelt.Blazor.HeadElement#9]
Starting with .NET 5 RC1, there's a binary-breaking change in `RenderTreeFrame` that will affect this library because of how it uses the internal `RenderTreeFrame` APIs. For this library to work with .NET 5 RC1 and later, you'll need to recompile. And because you're using the `Microsoft.AspNetCore.Components.RenderTree.*` APIs directly you'll have to multitarget `netstandard2.0` and `net5.0` if you want to retain compatibility with both.

I think [this line here](https://github.com/jsakamoto/Toolbelt.Blazor.HeadElement/blob/master/Components/HeadElement.csproj#L4) would need to be changed to:

```xml
<TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
```

... and then instead of depending purely on `Microsoft.AspNetCore.Components.Web` 3.0.0 directly, you'll need to depend on either the 3.x or 5.x versions depending on target framework. For example, replace [this line here](https://github.com/jsakamoto/Toolbelt.Blazor.HeadElement/blob/master/Components/HeadElement.csproj#L30) with something like:

```xml
<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="3.0.0" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="5.0.0-rc.1.*" Condition="'$(TargetFramework)' != 'netstandard2.0'" />
```

**Comment by jsakamoto** [comment]
@SteveSandersonMS Thank you for your reporting!

And, I appreciate your very kind comment. 👍 

Because of your comment in this thread, I could reproduce the problem, and I could fix it at commit 090df430 in the `net5.0` branch in this repository.

Finally, I published the latest fixed version of the `HeadElement` NuGet package.

- https://www.nuget.org/packages/Toolbelt.Blazor.HeadElement/5.0.0-rc.1

Again, thank you for your contribution!

**Comment by SteveSandersonMS** [comment]
Also thanks to you @jsakamoto for producing this package!
